skate-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -1,34 +1,58 @@
+         Simple HTML elements (text formatting, paragraphs, and lists) can be added inside of <![CDATA[ ]]> tag.
+         Guidelines: https://plugins.jetbrains.com/docs/marketplace/plugin-overview-page.html#plugin-description -->
+    <description><![CDATA[
     A plugin for IntelliJ and Android Studio for faster Kotlin and Android development!
   ]]></description>
 
-  <!-- Product and plugin compatibility requirements.
-       Read more: https://plugins.jetbrains.com/docs/intellij/plugin-compatibility.html -->
-  <depends>com.intellij.modules.platform</depends>
+    <!-- Product and plugin compatibility requirements.
+         Read more: https://plugins.jetbrains.com/docs/intellij/plugin-compatibility.html -->
+    <depends>com.intellij.modules.platform</depends>
 
-  <!-- Extension points defined by the plugin.
-       Read more: https://plugins.jetbrains.com/docs/intellij/plugin-extension-points.html -->
-  <extensions defaultExtensionNs="com.intellij">
+    <!-- Extension points defined by the plugin.
+         Read more: https://plugins.jetbrains.com/docs/intellij/plugin-extension-points.html -->
+    <!--    <extensions defaultExtensionNs="com.intellij">-->
 
-  </extensions>
-  <actions>
-    <action id="com.slack.sgp.intellij.DemoAction" class="com.slack.sgp.intellij.DemoAction" text="Demo Action"
-            description="The most descriptive description">
-    </action>
-  </actions>
+    <!--    </extensions>-->
+
+    <!--    <extensions defaultExtensionNs="com.android.tools.idea.assistant">-->
+    <!--        <assistantBundleCreator-->
+    <!--                id="HelloWorldBundle"-->
+    <!--                class="com.slack.sgp.intellij.sidepanel.impl.HelloWorldAssistantBundleCreator"/>-->
+    <!--    </extensions>-->
+
+    <!--    <actions>-->
+    <!--&lt;!&ndash;        <action id="com.slack.sgp.intellij.DemoAction" class="com.slack.sgp.intellij.sidepanel.DemoAction" text="Demo Action"-->
+    <!--                description="The most descriptive description">-->
+    <!--        </action>&ndash;&gt;-->
+    <!--        <group-->
+    <!--            id = "Hello World assistant"-->
+    <!--            description="Show it to the world!"-->
+    <!--        >-->
+    <!--            <action id="com.slack.sgp.intellij.OpenHelloWorldAssistant"-->
+    <!--                            class="com.slack.sgp.intellij.sidepanel.OpenAssistSidePanelAction"-->
+    <!--                            text="Open Hello World Assistant"-->
+    <!--                            description="Action to open the hello world side panel">-->
+    <!--                        <add-to-group group-id="ToolsMenu" anchor="last"/>-->
+    <!--                    </action>-->
+    <!--        </group>-->
+
+    <!--    </actions>-->
+
+    <extensions defaultExtensionNs="com.intellij">
+        <toolWindow id="What's New!" secondary="true" icon="AllIcons.Toolwindows.WebToolWindow" anchor="right"
```

Comment:
huh, this is actually way simpler than the Studio stuff we were copying. I like it 👍 

Though I think we need to do this dynamically so we can open it on demand